packages/editor/src/components/page-attributes/test/order.js

```diff
@@ -1,100 +1,93 @@
 /**
  * External dependencies
  */
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import { PageAttributesOrder } from '../order';
 
 describe( 'PageAttributesOrder', () => {
-	it( 'should reject invalid input', () => {
-		const onUpdateOrder = jest.fn();
-		const wrapper = mount(
-			<PageAttributesOrder onUpdateOrder={ onUpdateOrder } />
-		);
-
-		wrapper.find( 'input' ).simulate( 'change', {
-			target: {
-				value: 'bad',
-			},
-		} );
-
-		wrapper.find( 'input' ).simulate( 'change', {
-			target: {
-				value: '----+++',
-			},
-		} );
-
-		wrapper.find( 'input' ).simulate( 'change', {
-			target: {
-				value: '-',
-			},
+	/**
+	 * When starting to type inside the spinbutton, select the current value
+	 * in order to override it with the new value afterwards.
+	 */
+	const typeOptions = {
+		initialSelectionStart: 0,
+		initialSelectionEnd: 1,
+	};
```

Comment:
I know I'm late, but wouldn't this only the first character in the input's value, instead of the whole value?

A good alternative could be to use [the `clear` utility](https://testing-library.com/docs/user-event/utility#clear)

Reply:
I did try `.clear()` but didn't like the end result, because it was adding additional invocations of the mocks, while this approach kept them the same as the original.